content/installation/rust.md

```diff
@@ -6,4 +6,147 @@ implementation: unofficial
 weight: 1200
 ---
 
-Please see the [Cucumber-Rust website](https://github.com/bbqsrc/cucumber-rust).
+## Cucmber-Rust
+
+Cucumber-Rust is an unofficial implementation of the Cucumber testing framework for Rust. Fully native, no external test runners or dependencies.
+
+### Usage
+
+Create a directory called `tests/` in your project root and create a test target of your choice. In this example we will name it `cucumber.rs`.
+
+Add this to your `Cargo.toml`:
+
+```toml
+[[test]]
+name = "cucumber"
+harness = false # Allows Cucumber to print output instead of libtest
+
+[dev-dependencies]
+cucumber_rust = "^0.5.1"
+```
+
+If using stable Rust, you need to disable the `nightly` feature:
+
+```
+cucumber_rust = { version = "^0.5.1", default-features = false }
+```
+
+Create a directory called `features/` and put a feature file in it named something like `example.feature`. It might look like:
+
+```gherkin
+Feature: Example feature
+
+  Scenario: An example scenario
+    Given I am trying out Cucumber
+    When I consider what I am doing
+    Then I am interested in ATDD
```

Comment:
Is there a reason you are referring to ATDD instead of BDD?

Reply:
This is example from official repo

Reply:
OK, thanks :)